api/.env.example

```diff
@@ -30,3 +30,6 @@ DOI_PREFIX=10.82259
 DATACITE_ENDPOINT=https://api.test.datacite.org/dois
 DATACITE_USER=user
 DATACITE_PASSWORD=password
+
+MAIL_SERVER=localhost
```

Comment:
MAIL_SERVER needs to be 0.0.0.0 on localhost. Email verification doesn't work with MAIL_SERVER=localhost